File: tests/fixtures/sample.php
<?php

use App\Http\Controllers\UserController;
use Illuminate\Support\Facades\DB;

class Calculator
{
    private float $value;

    public function __construct(float $value)
    {
        $this->value = $value;
    }

    public function add(float $x): float
    {
        return $this->value + $x;
    }

    public function subtract(float $x): float
    {
        return $this->value - $x;
    }
}

$calc = new Calculator(10.0);
$result = $calc->add(5.0);
echo $result;

File: tests/fixtures/sample.cs
using System;
using System.Collections.Generic;

namespace MyApp
{
    public class Calculator
    {
        private double value;

        public Calculator(double value)
        {
            this.value = value;
        }

        public double Add(double x)
        {
            return this.value + x;
        }

        public static void Main(string[] args)
        {
            var calc = new Calculator(10.0);
            double result = calc.Add(5.0);
            Console.WriteLine(result);
        }
    }
}
